js/viewer/attributes.js

```diff
@@ -0,0 +1,48 @@
+/*globals */
+
+OC.Plugins.register('OC.Share.ShareItemModel', {
+	attach: function (model) {
+
+		if (_.isUndefined(model)) {
+			throw 'missing OC.Share.ShareItemModel';
+		}
+
+		// Make can-download available permission as checkbox
+		var mimetype = model.getFileInfo().getMimeType();
+		var folderMimeType = 'httpd/unix-directory';
+		if (odfViewer.isSupportedMimeType(mimetype) || mimetype === folderMimeType) {
+			// With read-only permission set for the file, download will be disabled. Only viewing will be allowed
+			$.get(
+				OC.filePath('richdocuments', 'ajax', 'settings.php'),
```

Comment:
@mrow4a why is this call to the app needed? All share attributes should already be in the model - right?

Reply:
note: you can expose app settings using the config.php hook: https://github.com/owncloud/core/blob/stable10/core/js/config.php#L222 and an example here https://github.com/owncloud/core/blob/stable10/apps/files/appinfo/app.php#L65